core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -2456,4 +2471,24 @@ private CarbonCommonConstants() {
    * property which defines the insert stage flow
    */
   public static final String IS_INSERT_STAGE = "is_insert_stage";
+
+  /**
+   * the level 1 complex delimiter default value
+   */
+  @CarbonProperty
```

Comment:
This looks to be just a value. Not the user configuration property. If so, @CarbonProperty is not required. please check and remove

Reply:
Removed CarbonProperty